core/server/api/v2/utils/validators/input/schemas/tags.json

```diff
@@ -0,0 +1,63 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "tags",
+    "title": "tags",
+    "description": "Base tags definitions",
+    "definitions": {
+        "tag": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "maxLength": 191,
+                    "pattern": "^([^,]|$)"
+                },
+                "slug": {
+                    "type": ["string", "null"],
```

Comment:
What does that mean? :)

Reply:
This allows creating a tag with `null` in a slug (that's current behavior in Ghost-Admin). The current behavior is such that when sending `slug:null` the slug is being regenerated to a new value.

Reply:
Is "tag.name" required?

Reply:
It is required for `.add` action - https://github.com/TryGhost/Ghost/pull/10486/files#diff-9442db3b9160693912358aa5f5cc1bf7R18

Reply:
👍